besu/src/test/resources/everything_config.toml

```diff
@@ -101,6 +101,7 @@ miner-stratum-enabled=false
 miner-coinbase="0x0000000000000000000000000000000000000002"
 miner-extra-data="0x444F4E27542050414E4943202120484F444C2C20484F444C2C20484F444C2021"
 min-gas-price=1
+min-block-occupancy-ratio=0.7
```

Comment:
the ratio should not be 0.8 in this conf?

Reply:
Usually we don't use the default value so that we are sure the config file is correctly loaded

Reply:
Ok